lib/fb-submitter-client.unit.spec.js

```diff
@@ -40,17 +40,17 @@ const expectedEncryptedData = 'pOXXs5YW9mUW1weBLNawiMRFdk6Hh92YBfGqmg8ych8PqnZ5l
  *
  **/
 const testInstantiation = (t, params, expectedCode, expectedMessage) => {
-  t.plan(4)
-
   let failedClient
   try {
     failedClient = new FBSubmitterClient(...params)
+    t.notOk('it should throw an error and not get here')
```

Comment:
I think the semantics of '.throws()' is nicer but just a small preference

Reply:
ah yes - now I remember

Reply:
Done

Reply:
Thanks 👍

Reply:
Just checked the code, think it should look more like this?

```
  const error = t.throws(() => fnWhichThrows(), 'should throw an error')
  t.equal(error.name, 'TheErrorName', '...')
  t.equal(error.data.thing, 'yes', '...')
```

As in is better to use one or the other, but not both